addons/test_mail/models/test_mail_corner_case_models.py

```diff
@@ -216,6 +216,12 @@ class MailTestMultiCompanyWithActivity(models.Model):
     company_id = fields.Many2one("res.company")
 
 
+class MailTestMultiCompanyWithActivityAdmin(MailTestMultiCompanyWithActivity):
```

Comment:
Instead of a class, can't you locally in the test patch check_access_rights ? It is already done in some mail tests, when we want a specific test but don't want to add a model just for that :)